src/fe/explorer.js

```diff
@@ -0,0 +1,1117 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+//
+// You may not use this file except in compliance with the License. A copy
+// of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+// either express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+var s3explorer_columns = { check:0, object:1, folder:2, date:3, timestamp:4, storageclass:5, size:6 };
+
+// Cache frequently-used selectors and data table
+var $tb = $('#s3objects-table');
+var $bc = $('#breadcrumb');
+var $bl = $('#bucket-loader');
+let hardcoded_settings = {
+            "auth": "auth",
+            "region": "",
+            "bucket": "bucket-1",
+            "entered_bucket": "bucket-1",
+            "selected_bucket": "",
+            "view": "folder",
+            "delimiter": "/",
+            "prefix": "",
+            "mfa": {
+                "use": "no",
+                "code": ""
+            },
+            "cred": {
+                "accessKeyId": "AKIAIOSFODNN7EXAMPLE",
```

Comment:
We can't hard-code credentials like this, so we have to figure something out if we want to keep the FE code in here. The FE should probably prompt the user for credentials (easiest way) or we _could_ feed them in from the Wasmer FE in some way (convenience feature, not quite necessary if you ask me)

Reply:
yeah prompting the user for creds is the best, lets do that!